src/app/pages/HomePage/index.tsx

```diff
@@ -85,9 +88,12 @@ export const HomePage: FC = () => {
         <LogotypeBox>
           <Logotype showText />
         </LogotypeBox>
-        <SearchInputBox>
-          <Search onFocusChange={onFocusChange} />
-        </SearchInputBox>
+        <Box sx={{ position: 'relative', zIndex: 2 }}>
+          <SearchInputBox>
+            <Search disabled={isApiOffline} onFocusChange={onFocusChange} />
+          </SearchInputBox>
+          {isApiOffline && <OfflineIndicator />}
+        </Box>
         <Box sx={{ zIndex: 1 }}>
```

Comment:
Do we need z-index on ParaTimeSelector wrapper ?

Reply:
Actually it is a way of "stacking" layers in the opposite order, as search bar can sometimes overlap with ParaTime selector.